### DISCUSSION TITLE
I get 'Android Device detected, signing default bug flags' when pluggin in an android device via USB - but what does that mean, how do I use it?

### DISCUSSION THREAD

**Discussion by emilBeBri** [gokcehan/lf#1493] [Q&A] [answered]
So as the title says - I've tried looking at the FAQ, the documentation, searching the discussion - what does it mean, and how do I utilize it? Right Now I have a shell script that mounts my android device but if LF already does it, and probably better than my makeshift shellscripting, that would be very nice.

**Comment by DusanLesan** [comment, marked as answer]
That looks like `libmtp` log entry. Lf does not do any such actions (I think)
If you see that message inside of `lf`, you are probably calling the script from `lf` and it is catching the `libmtp` output (or from whatever tool that is using  libmtp)
